### DISCUSSION TITLE
Option for AssertModelIs - where to put it

### DISCUSSION THREAD

**Discussion by powellblyth** [laravel/framework#38483] [Q&A]
Hi there, 

I wrote for my project, in the base TestCase class, a little helper to do `AssertModelIs` and `AssertModelIsNot`

instead of a slightly ugly
```php
$this->assertTrue($expectedModel->is($otherModel), 'The wrong model was passed'));
```
I wanted
```php
$this->assertModelIs($expectedModel, $otherModel);
```

which is a bit more readable.

I wondered where to put it, so that I could put a PR in. but I didn't find a home.

I see that `Illuminate\Foundation\Testing\Concerns` has some traits in it, but `Illuminate\Foundation\Testing\Concerns\InteractsWithModels` doesn't quite feel right

I didn't want to put it in the base model, since obviously the model isn't necessarily being mocked.

Does anyone have time to make a suggestion where might be the correct place to add this pair of assertions please?

**Comment by crynobone** [comment]
You should be able to use `Tests\TestCase` for this purpose.
